final_task/rss_reader/requirements.txt

```diff
@@ -0,0 +1,4 @@
+feedparser==5.2.1
+argparse==1.4.0
+json==2.0.9
```

Comment:
json is a standart library, no need to include it in this file

Reply:
Done